### PR TITLE
feat: add isElectronMain env test

### PR DESCRIPTION
**Pull request by hugomrdias**

### CODE REVIEW

File: test/env.spec.js [hunk 1]
```diff
@@ -27,13 +30,37 @@ describe('env', function () {
     }
   })
 
+  it('isElectronMain should have the correct value in each env', function () {
+    if (env.isElectron && !env.isElectronRenderer) {
+      return expect(env.isElectronMain).to.be.true()
+    }
+    if (env.isElectronMain) {
+      return expect(env.isElectronMain).to.be.true()
```
Comment (achingbrain) [inline test/env.spec.js]: This only asserts that `env.isElectronMain` is true when it is true (or truthy, at least).  Perhaps it should assert that `env.isElectron` is true and `env.isElectronRenderer` is false when `env.isElectronMain` is true?

It also needs to assert that we actually hit some these of assertions when the test is run.

Comment (hugomrdias) [reply test/env.spec.js]: added the throw at the end, this way that code makes more sense.

File: test/env.spec.js [hunk 2]
```diff
@@ -10,92 +10,140 @@ const expect = chai.expect
...
+        expect(env.isElectron).to.be.false()
+        break
+      case 'browser':
+        expect(env.isElectron).to.be.false()
+        break
+      case 'webworker':
+        expect(env.isElectron).to.be.false()
+        break
+      default:
+        expect.fail(`Could not detect end. Current env is ${process.env.AEGIR_RUNNER}`)
```
Comment (achingbrain) [inline test/env.spec.js]:
```suggestion
        expect.fail(`Could not detect env. Current env is ${process.env.AEGIR_RUNNER}`)
```

File: test/env.spec.js [hunk 3]
```diff
@@ -10,92 +10,140 @@ const expect = chai.expect
...
+        expect(env.isElectronMain).to.be.false()
+        break
+      case 'browser':
+        expect(env.isElectronMain).to.be.false()
+        break
+      case 'webworker':
+        expect(env.isElectronMain).to.be.false()
+        break
+      default:
+        expect.fail(`Could not detect end. Current env is ${process.env.AEGIR_RUNNER}`)
```
Comment (achingbrain) [inline test/env.spec.js]:
```suggestion
        expect.fail(`Could not detect env. Current env is ${process.env.AEGIR_RUNNER}`)
```

File: test/env.spec.js [hunk 4]
```diff
@@ -10,92 +10,140 @@ const expect = chai.expect
...
+        expect(env.isElectronRenderer).to.be.false()
+        break
+      case 'browser':
+        expect(env.isElectronRenderer).to.be.false()
+        break
+      case 'webworker':
+        expect(env.isElectronRenderer).to.be.false()
+        break
+      default:
+        expect.fail(`Could not detect end. Current env is ${process.env.AEGIR_RUNNER}`)
```
Comment (achingbrain) [inline test/env.spec.js]:
```suggestion
        expect.fail(`Could not detect env. Current env is ${process.env.AEGIR_RUNNER}`)
```

File: test/env.spec.js [hunk 5]
```diff
@@ -10,92 +10,140 @@ const expect = chai.expect
...
+        expect(env.isNode).to.be.true()
+        break
+      case 'browser':
+        expect(env.isNode).to.be.false()
+        break
+      case 'webworker':
+        expect(env.isNode).to.be.false()
+        break
+      default:
+        expect.fail(`Could not detect end. Current env is ${process.env.AEGIR_RUNNER}`)
```
Comment (achingbrain) [inline test/env.spec.js]:
```suggestion
        expect.fail(`Could not detect env. Current env is ${process.env.AEGIR_RUNNER}`)
```

File: test/env.spec.js [hunk 6]
```diff
@@ -10,92 +10,140 @@ const expect = chai.expect
...
+        expect(env.isBrowser).to.be.false()
+        break
+      case 'browser':
+        expect(env.isBrowser).to.be.true()
+        break
+      case 'webworker':
+        expect(env.isBrowser).to.be.false()
+        break
+      default:
+        expect.fail(`Could not detect end. Current env is ${process.env.AEGIR_RUNNER}`)
```
Comment (achingbrain) [inline test/env.spec.js]:
```suggestion
        expect.fail(`Could not detect env. Current env is ${process.env.AEGIR_RUNNER}`)
```

File: test/env.spec.js [hunk 7]
```diff
@@ -10,92 +10,140 @@ const expect = chai.expect
...
+        expect(env.isWebWorker).to.be.false()
+        break
+      case 'browser':
+        expect(env.isWebWorker).to.be.false()
+        break
+      case 'webworker':
+        expect(env.isWebWorker).to.be.true()
+        break
+      default:
+        expect.fail(`Could not detect end. Current env is ${process.env.AEGIR_RUNNER}`)
```
Comment (achingbrain) [inline test/env.spec.js]:
```suggestion
        expect.fail(`Could not detect env. Current env is ${process.env.AEGIR_RUNNER}`)
```

**Pull request merged**